Cython/Compiler/UtilNodes.py

```diff
@@ -155,10 +155,12 @@ def analyse_types(self, env):
         return self
 
     def infer_type(self, env):
-        if self.type is not None:
+        # AttributeNode (and possibly others) have ErrorType by default, so make sure to infer
+        # if the result is an error, rather than just accepting it
```

Comment:
… maybe it's not a good idea that they have `ErrorType` by default? Should we change that instead?